src/platform/forms/tests/save-in-progress/SaveStatus.unit.spec.jsx

```diff
@@ -84,4 +84,29 @@ describe('<SaveStatus>', () => {
     );
     expect(tree.subTree('.panel').text()).to.include('ID number is 98765');
   });
+  it('should display the appSavedSuccessfullyMessage with custom app type & SiP ID', () => {
+    const appSavedSuccessfullyMessageProps = {
+      ...props,
+      form: {
+        formId: VA_FORM_IDS.FORM_10_10EZ,
+        lastSavedDate: 1505770055,
+        autoSavedStatus: 'success',
+        loadedData: {
+          metadata: {
+            inProgressFormId: 98765,
+          },
+        },
+      },
+      formConfig: {
+        customText: {
+          appType: 'custom application type',
+        },
+      },
+    };
+    const tree = SkinDeep.shallowRender(
+      <SaveStatus {...appSavedSuccessfullyMessageProps} />,
+    );
+    expect(tree.subTree('.panel').text()).to.include('custom application type');
```

Comment:
I don't think this test is specific enough. The `.panel` div would contain:

> custom application type has been saved. It was last saved on [XYZ]. Your custom application type ID number is 98765.

Since the `appType` appears in multiple places, we should be specifically checking that we find "Your custom application type ID number is 98765", otherwise this test could be giving false positives.

Reply:
Good insight @bkjohnson! I updated the tests :-)